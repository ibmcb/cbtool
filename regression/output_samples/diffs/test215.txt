 status: Waiting for vm_4 (cloud-assigned uuid 2C2DAAA4-9C26-56B5-B4E6-CE10ABE5C128) to start...
 status: Trying to establish network connectivity to vm_4 (cloud-assigned uuid 2C2DAAA4-9C26-56B5-B4E6-CE10ABE5C128), on IP address 117.87.103.220...
 status: Bypassing the sending of a copy of the code tree to vm_4 (117.87.103.220)...
 status: Bypassing generic VM post_boot configuration on vm_4 (117.87.103.220)...
VM object F09577B3-3400-54A7-A520-21B2C188F40C (named "vm_4") sucessfully attached to this experiment. It is ssh-accessible at the IP address 117.87.103.220 (2C2DAAA4-9C26-56B5-B4E6-CE10ABE5C128.simcloud.com).
|vm_1          |tinyvm              |pico32      |223.57.181.60   |simhostb3                  |SUT                 |none    |none       |871DBCA3-6BD3-51CC-AD1C-EC170141AF08
|vm_2          |tinyvm              |pico32      |135.191.127.236 |simhostd1                  |LG                  |none    |none       |78DA3EBC-2340-555E-B102-0F1F2796FC6E
|vm_3          |tinyvm              |pico32      |212.135.120.189 |simhostb1                  |SUT                 |none    |none       |A2868A5A-49AD-5DD7-A350-BD5C2807154B
|vm_4          |db2                 |copper32    |117.87.103.220  |simhostc0                  |SUT                 |none    |none       |F09577B3-3400-54A7-A520-21B2C188F40C
|host_name                               |simhostc0                            
|vmc_name                                |simzone_c                            
|Used Memory                                          |19.06M                        
|Uptime (in seconds)                                  |2516390                       
|Total Connections Received                           |185609                        
|Total Commands Processed                             |11570399                      
|Storage Size                                         |18350080                      
|Data Size                                            |6567308                       
|Index Size                                           |2068528                       
|Average Object Size                                  |614.571214673                 
|AD93E38D-EAEC-5C15-BA20-7A27F63D13FF (simzone_b) VM RESERVATIONS|2                             
|E4B581D8-2DD3-5956-8196-B1B60C5DC5B0 (simzone_d) VM RESERVATIONS|1                             
|AD29082C-1389-58A3-BE64-3784D075F191 (simzone_c) VM RESERVATIONS|1                             
|75A875C6-F2C5-5C3D-BD34-051FBBE24C3A (simzone_a) VM RESERVATIONS|0                             
