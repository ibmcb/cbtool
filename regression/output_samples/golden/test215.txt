 status: Waiting for vm_4 (cloud-assigned uuid 300AC378-C0AF-5610-808B-F14A9F1EC5F4) to start...
 status: Trying to establish network connectivity to vm_4 (cloud-assigned uuid 300AC378-C0AF-5610-808B-F14A9F1EC5F4), on IP address 4.250.206.6...
 status: Bypassing the sending of a copy of the code tree to vm_4 (4.250.206.6)...
 status: Bypassing generic VM post_boot configuration on vm_4 (4.250.206.6)...
VM object 38883899-F78B-51A4-952C-63B170E0F5C8 (named "vm_4") sucessfully attached to this experiment. It is ssh-accessible at the IP address 4.250.206.6 (300AC378-C0AF-5610-808B-F14A9F1EC5F4.simcloud.com).
The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |113.203.60.155  |simhosta3                  |SUT                 |none    |none       |A71367A7-681B-5FEA-94C1-285EF7566DDD
|vm_2          |tinyvm              |pico32      |163.115.104.223 |simhostd2                  |LG                  |none    |none       |30405394-CB90-5A4E-8132-D31A65A9CC4D
|vm_3          |tinyvm              |pico32      |249.146.224.254 |simhostb1                  |SUT                 |none    |none       |38C75BAA-74EF-50E0-B26B-56B98CA05C7C
|vm_4          |db2                 |copper32    |4.250.206.6     |simhosta2                  |SUT                 |none    |none       |38883899-F78B-51A4-952C-63B170E0F5C8

The VM object vm_4, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|host_name                               |simhosta2                            
|meta_tags                               |{'a': 'b', 'x': 'y', 'r': '2'}       
|size                                    |copper32                             
|vmc_name                                |simzone_a                            
|vmc_pool                                |SUT                                  

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |19.13M                        
|Redis Version                                        |2.6.2                         
|Uptime (in seconds)                                  |2511677                       
|Total Connections Received                           |185331                        
|Total Commands Processed                             |11050543                      
|Number of Keys                                       |313                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |18321408                      
|Data Size                                            |5826476                       
|Index Size                                           |1888656                       
|Average Object Size                                  |620.431902886                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |4                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |4                             
|VMs ARRIVED                                          |4                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |92                            

 ---------------- VMC-WIDE COUNTERS ----------------
|C4E90296-1043-55C8-8270-B5D7C3C38270 (simzone_b) VM RESERVATIONS|1                             
|2AEE56D4-E6A4-5FB2-8D29-4268D4C22AD0 (simzone_a) VM RESERVATIONS|2                             
|5F641DA9-DE4B-538C-B0CD-42A4F7D43E9E (simzone_d) VM RESERVATIONS|1                             
|A796D59F-B618-5C91-8FC6-F6F093DE22C5 (simzone_c) VM RESERVATIONS|0                             

